samples/sensor/bh1750/Kconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020, Nordic Semiconductor ASA
+# Copyright (c) 2022, morsisko
+# SPDX-License-Identifier: Apache-2.0
+
+config LOG
+	default y
+
+config LOG_PRINTK
+	default y
+
+config SENSOR_LOG_LEVEL
+	default 4
+
+config I2C
+	default y
```

Comment:
please use `prj.conf`, this file is redundant since it does not define any Kconfig option.